index.html

```diff
@@ -24,9 +24,9 @@ <h3 class="poster-quote"></h3>
       <h2>Create your own motivational poster</h2>
       <label for="poster-image-url">Image url</label>
       <input type="text" name="poster-image-url" id="poster-image-url" placeholder="https://gph.is/2n553Ra">
-      <label for="poster-title">Motivational poster-title</label>
+      <label for="poster-title">Motivational title</label>
       <input type="text" name="poster-title" id="poster-title" placeholder="Growth Mindset">
-      <label for="poster-quote">Motivational poster-quote</label>
+      <label for="poster-quote">Motivational quote</label>
         <input type="text" name="poster-quote" id="poster-quote" placeholder="Hang in there!">
       <button class="make-poster">Show my poster</button>
     </form>
```

Comment:
lines 41-42: Looks like there's only two javascript files right now that you are importing into the template. Could `main.js` be broken up into a two are three separate files, in a logical way?